analyzers/tests/SonarAnalyzer.TestFramework.Test/SonarAnalyzer.TestFramework.Test.csproj

```diff
@@ -0,0 +1,28 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>net8.0-windows</TargetFramework>
```

Comment:
why targetting this here again for my own understanding.

Reply:
Same as for the main test project. Anything that references project with windows reference, needs to target windows.